scss/_type.scss

```diff
@@ -115,7 +115,7 @@ mark,
 .blockquote-footer {
   display: block;
   font-size: 80%; // back to default font-size
-  color: $blockquote-small-color;
+  filter: opacity($blockquote-footer-opacity);
```

Comment:
Only technical comment :)
I would use `opacity` instead of `filter:opacity`

Reply:
Hi @wojtask9 , you are absolutely right. I just like `filter` better but in this case it's a matter of taste. I'll change it to `opacity` and update the PR.

Thanks a lot for the review.